dipy/workflows/reconst.py

```diff
@@ -504,6 +506,185 @@ def run(self, input_files, bvalues, bvectors, mask_files,
 
             return io_it
 
-    def get_tensor_model(self, gtab):
-        return TensorModel(gtab, fit_method="RT", sigma=self.sigma,
-                           jac=self.jacobian)
+
+class ReconstDkiFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'dki'
+
+    def run(self, input_files, bvalues, bvectors, mask_files, b0_threshold=0.0,
+            save_metrics=[],
+            out_dir='', out_dt_tensor='dti_tensors.nii.gz', out_fa='fa.nii.gz',
+            out_ga='ga.nii.gz', out_rgb='rgb.nii.gz', out_md='md.nii.gz',
+            out_ad='ad.nii.gz', out_rd='rd.nii.gz', out_mode='mode.nii.gz',
+            out_evec='evecs.nii.gz', out_eval='evals.nii.gz',
+            out_dk_tensor="dki_tensors.nii.gz",
+            out_mk="mk.nii.gz", out_ak="ak.nii.gz", out_rk="rk.nii.gz"):
+        """ Workflow for Diffusion Kurtosis reconstruction and for computing
+        DKI metrics. Performs a DKI reconstruction on the files by 'globing'
+        ``input_files`` and saves the DTI metrics in a directory specified by
+        ``out_dir``.
+
+        Parameters
+        ----------
+        input_files : string
+            Path to the input volumes. This path may contain wildcards to
+            process multiple inputs at once.
+        bvalues : string
+            Path to the bvalues files. This path may contain wildcards to use
+            multiple bvalues files at once.
+        bvectors : string
+            Path to the bvalues files. This path may contain wildcards to use
+            multiple bvalues files at once.
+        mask_files : string
+            Path to the input masks. This path may contain wildcards to use
+            multiple masks at once. (default: No mask used)
+        b0_threshold : float, optional
+            Threshold used to find b=0 directions (default 0.0)
+        save_metrics : variable string, optional
+            List of metrics to save.
+            Possible values: fa, ga, rgb, md, ad, rd, mode, tensor, evec, eval
+            (default [] (all))
+        out_dir : string, optional
+            Output directory (default input file directory)
+        out_tensor : string, optional
```

Comment:
Unexpected parameter `out_tensor` in docstring. The function `run` has  `out_dt_tensor` and `out_dk_tensor`